src/tests/ftest/server/daos_server_config.yaml

```diff
@@ -169,3 +169,8 @@ server_config_val: !mux
       - "targets"
       - "ABDCD"
       - "FAIL"
+  targets_str_negative:
```

Comment:
Minor nit, but this was previously called `targets_negative`

Reply:
done.